vendor/google.golang.org/grpc/stream.go

```diff
@@ -149,7 +149,7 @@ type clientStream struct {
 
 	tracing bool // set to EnableTracing when the clientStream is created.
 
-	mu sync.Mutex
```

Comment:
@corecode this is go fmt plugin change, i just notice it is in vendor catelog, did it need change?